example/Components/SimpleForm.js

```diff
@@ -28,27 +28,47 @@ const Form = ({
 (
   <div className='form'>
     <div className='formGroup'>
-      <label>Name</label>
+      <label htmlFor='name'>Name</label>
       <input
         type='text'
+        id='name'
+        aria-errormessage='name-error'
```

Comment:
`aria-errormessage` uses an `id`, much like`for`, to link the input to its error message.